ui/src/app/tokens/Tokens.tsx

```diff
@@ -131,13 +131,9 @@ export default function Tokens() {
           panelType="Tokens"
           setOpen={setShowDeleteTokenModal}
           handleDelete={() =>
-            deleteTokens(deletingTokens?.map((t) => t.id) || []).then(() => {
-              incrementTokensVersion();
-            })
+            deleteTokens(deletingTokens?.map((t) => t.id) || [])
           }
-          onSuccess={() => {
-            incrementTokensVersion();
-          }}
+          onSuccess={() => {}}
```

Comment:
we could make this optional probably, the `onSuccess` prop?

Reply:
In fact, it is optional. I will fix it